### PR TITLE
chore(Makefile): Adding unused package check for the chaos-operator

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj Das <raj.das@mayadata.io>

**What this PR does / why we need it**:
Adding unused package check for the chaos-operator

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag
-   [ ] PR messages has document related information
-   [ ] Labelled this PR & related issue with `breaking-changes` tag
-   [ ] PR messages has breaking changes related information
-   [ ] Labelled this PR & related issue with `requires-upgrade` tag
-   [ ] PR messages has upgrade related information
-   [ ] Commit has unit tests
-   [ ] Commit has integration tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,24 +63,33 @@ lint:
...
+dockerops:
 	@echo "------------------"
-	@echo "--> Build & Push chaos-operator docker image" 
+	@echo "--> Build & Push chaos-operator docker image"
 	@echo "------------------"
 	sudo docker build . -f build/Dockerfile -t $(DOCKER_REPO)/$(DOCKER_IMAGE):$(DOCKER_TAG)
-	REPONAME=$(DOCKER_REPO) IMGNAME=$(DOCKER_IMAGE) IMGTAG=$(DOCKER_TAG) ./buildscripts/push
\ No newline at end of file
+	REPONAME=$(DOCKER_REPO) IMGNAME=$(DOCKER_IMAGE) IMGTAG=$(DOCKER_TAG) ./buildscripts/push
+
```
Comment (rahulchheda) [inline Makefile:87]:
```suggestion
.PHONY: unused-package-check
```

Comment (rajdas98) [reply Makefile:87]: unused-package-check call from `deps` in line no. 27

Comment (rahulchheda) [reply Makefile:87]: What if someone wants to call it without calling `deps`?

Comment (rajdas98) [reply Makefile:87]: You can call directly `make unuse-package-check`

Comment (rahulchheda) [reply Makefile:87]: Good to go then

**Pull request merged**